app/helpers/drawing_helper.rb

```diff
@@ -8,4 +8,8 @@ def filter_archived(drawings)
   def filter_current(drawings)
     drawings.select { |plan| plan.archived? == false }.sort_by(&:archived_at)
   end
+
+  def archive_reason_collection_for_radio_buttons
+    Drawing.archive_reasons.keys.map { |k| [k, I18n.t(k)] }
```

Comment:
I know that in the project we generally use `I18n.t` where we could use `t`. It's on my list of things we can improve. But since I bump into this again I think it would be great to start using just `t` which is a Rails helper method and also adds some extra features, which are not provided by the `I18n.t` method. Rails `t` helper method also adds security awareness through its built in XSS protection, which the `t` method has to deal with because it adds strings to the view.

Feel free to have a look at [this][1]

[1]: https://oananao.wordpress.com/2014/07/12/whats-the-difference-between-using-i18n-t-and-simply-t/

Reply:
Good idea - I have incorporated this into the Pivotal story pt-173627179.